hummingbot/strategy/cross_exchange_market_making/start.py

```diff
@@ -58,15 +64,26 @@ def start(self):
     maker_market_trading_pair_tuple = MarketTradingPairTuple(*maker_data)
     taker_market_trading_pair_tuple = MarketTradingPairTuple(*taker_data)
     self.market_trading_pair_tuples = [maker_market_trading_pair_tuple, taker_market_trading_pair_tuple]
-    self.market_pair = CrossExchangeMarketPair(maker=maker_market_trading_pair_tuple, taker=taker_market_trading_pair_tuple)
+    self.market_pair = MakerTakerMarketPair(maker=maker_market_trading_pair_tuple, taker=taker_market_trading_pair_tuple)
+
+    if debug_price_shim:
+        amm_connector: GatewayEVMAMM = cast(GatewayEVMAMM, taker_market)
+        GatewayPriceShim.get_instance().patch_prices(
+            maker_market,
+            maker_trading_pair,
+            amm_connector.connector_name,
+            amm_connector.chain,
+            amm_connector.network,
+            taker_trading_pair
+        )
```

Comment:
We should only do this if the taker market is a gateway connector

Reply:
✅